sdk/runanywhere-kotlin/src/jvmAndroidMain/kotlin/com/runanywhere/sdk/foundation/bridge/extensions/CppBridgeHTTP.kt

```
@@ -229,23 +239,129 @@ object CppBridgeHTTP {
+                    os.write(body)
+                    os.flush()
+                }
+            }
+
+            statusCode = connection.responseCode
+
+            val inputStream =
+                if (HttpStatus.isSuccess(statusCode)) connection.inputStream
+                else connection.errorStream
+
+            val responseBytes = inputStream?.use { it.readBytes() }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)